frontend/src/data/Breakdowns.ts

```diff
@@ -24,81 +24,80 @@ export const BREAKDOWN_VAR_DISPLAY_NAMES: Record<BreakdownVar, string> = {
   state_fips: "State FIPS Code",
 };
 
+interface DemographicBreakdown {
+  // Name of the column in the returned data
+  readonly columnName: BreakdownVar;
+  // Whether the demographic breakdown is requested
+  enabled: boolean;
+  // If requested, should the breakdown include a "total", i.e. value for all age/race/sex
+  includeTotal: boolean;
+}
+
+function stringifyDemographic(breakdown: DemographicBreakdown) {
+  if (breakdown === undefined || !breakdown.enabled) {
+    return undefined;
+  }
+  return breakdown.includeTotal ? "with total" : "without total";
+}
+
+function createDemographicBreakdown(
+  columnName: string,
+  enabled = false,
+  includeTotal = false
+) {
+  return {
+    columnName: columnName as BreakdownVar,
+    enabled: enabled,
+    includeTotal: includeTotal,
+  };
+}
+
 export class Breakdowns {
   geography: GeographicBreakdown;
   // We may want to extend this to an explicit type to support variants for
   // day/week/month/year.
   time: boolean;
-  race: boolean;
-  race_nonstandard: boolean;
-  age: boolean;
-  sex: boolean;
+  demographicBreakdowns: Record<string, DemographicBreakdown>;
```

Comment:
Given the number of places were we directly access "race"/"age"/etc on this, could we make this a structured type? I think you can do that while still making it a map that is iterable:
```
type DemographicBreakdowns = {
  race: DemographicBreakdown;
  age: DemographicBreakdown;
  ...
}
```

Reply:
I added a type as the key. We could always change this later

Reply:
I made a type for the key... make the whole thing a structured type was introducing a lot of typescript complaints... we can always revisit